### PR TITLE
Intercepts of a line calculation for 4 forms of the equation of a line.

### PR DESCRIPTION
**Pull request by bkshep**
See Issue #34 

Program to calculate the intercepts of a vertical line, a horizontal line, a line in slope-intercept form, or a line in point-slope form.

### CODE REVIEW

File: Algebra/InterceptsOfALine.java [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+
+		while (restart) {
+			restart = false; // reset boolean
+
+			try {
+
+				System.out.println("\nPlease input the value of a: ");
+				double a = scan.nextDouble();
+
+				System.out.printf("\nThe equation of the line you entered is: y = %f",a);
```
Comment (Kyle-P) [inline Algebra/InterceptsOfALine.java]: The equation of the line here should read: "x = " instead of "y = " because it is a vertical line.

**Comment by Kyle-P** [comment]
Hi @bkshep. Great work! Thanks so much for all the contributions. Just one tiny thing before merging commented above.

Thank you!

**Comment by bkshep** [comment]
Hi @Kyle-P. Thank you for catching this typo! I have fixed it.

Thanks!

**Comment by Kyle-P** [comment]
Hi @bkshep! Thanks so much for your contributions! Will merge your work now.

**Pull request merged**